We are almost, but not quite, ready to switch to drake master.  Drake upstream switched the directory layout around, so we need some changes to delphyne and delphyne-gui, which I have in branches https://github.com/ToyotaResearchInstitute/delphyne/tree/drake-master and https://github.com/ToyotaResearchInstitute/delphyne-gui/tree/drake-master .  Unfortunately, those changes aren't quite enough.  When trying to do the cmake configure step in delphyne, I get:

```
CMake Error at backend/CMakeLists.txt:63 (add_library):
  Target "simulation_runner_py" links to target "Boost::boost" but the target
  was not found.  Perhaps a find_package() call is missing for an IMPORTED
  target, or an ALIAS target is missing?
```

I'm guessing that somehow the cmake dependencies changed, but I'm not quite sure how to make it work.  I don't quite have time to look into that today, so we can either just do the cherry-pick of the commit you need for now, or you can look into the Boost problem if you have time.

--- a/WORKSPACE
+++ b/WORKSPACE
@@ -263,8 +263,8 @@ github_archive(
 github_archive(
     name = "pybind11",
     repository = "RobotLocomotion/pybind11",
-    commit = "ffcf754ae9e766632610975d22372a86a7b63014",
-    sha256 = "7cd6f4efb02bf9ae17eeb2afba68023af913e61ae76e8b4254203d0eec019525",  # noqa
+    commit = "48999b69bde29cdf8d616d4fbd3d6ab1c561027d",
+    sha256 = "2ea18adfb608948cab1b5978081dc8c318ed47573ccd66f1603a37fbdbfc56da",  # noqa
     build_file = "tools/workspace/pybind11/pybind11.BUILD.bazel",
 )
 

